pom.xml

```diff
@@ -116,6 +116,14 @@
         <module>presto-plugin-toolkit</module>
     </modules>
 
+    <repositories>
```

Comment:
Why is that needed? Are we planning to merge this upstream, or is it going to be maintained as a differentiator?